src/data/cross-filter-stack-adapter.js

```diff
@@ -0,0 +1,49 @@
+import {CrossFilterSimpleAdapter} from './cross-filter-simple-adapter';
+
```

Comment:
What is specific to crossfilter about this adapter? It looks like it just calls `group.all()` - I hope we are keeping that convention even when crossfilter is not used?